bigbluebutton-html5/imports/ui/components/recording/component.jsx

```diff
@@ -50,11 +56,21 @@ class RecordingComponent extends React.PureComponent {
     const {
       intl,
       recordingStatus,
+      recordingTime,
       amIModerator,
       closeModal,
       toggleRecording,
     } = this.props;
 
+    let title;
+
+    if (!recordingStatus) {
+      title = recordingTime >= 0 ? intl.formatMessage(intlMessages.resumeTitle)
```

Comment:
Shouldn't the check be for `recordingTime > 0` because if the time is <= 0 then it hasn't been started yet.

Reply:
The recordTime is initiates at 0 and is updated in intervals, that is why 0 has to be checked since that indicates that recording has already begun.

Reply:
I didn't realize it would start and stay at 0 for a bit.